com.wso2telco.dep.hub.creditapi/.project

```diff
@@ -5,8 +5,14 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
 	</buildSpec>
 	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 		<nature>org.wso2.developerstudio.eclipse.mavenmultimodule.project.nature</nature>
 	</natures>
```

Comment:
Agreed. Azham, I informed you that your changes should only be for payment API. I still find changes unrelated to payment API. Please resend the PR again.